setup.py

```diff
@@ -75,7 +75,6 @@ def run(self):
     },
     classifiers=[
         "Programming Language :: Python :: 3",
-        "License :: OSI Approved :: MIT License",
```

Comment:
Why this change?

Reply:
I'll split this change out since it isn't relevant here..

Reply:
Added in gh-202, this change can be dropped